src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -383,17 +435,33 @@ protected void deleteFile(File deletedFile, File dir, File targetDir) {
             info("Deleted file: " + targetFile.getAbsolutePath());
         }
     }
+    
+    protected void cleanTargetDir(File outputDirectory){
+        File[] fList = outputDirectory.listFiles();
+        if (fList != null) {
+            for (File file : fList) {
+                if (file.isFile() && file.getName().toLowerCase().endsWith(".class")) {
+                   file.delete();
+                   info("Deleted Java class file: " + file);
+                } else if (file.isDirectory()) {
+                    cleanTargetDir(file);
+                }
+            }
+        }
+        
```

Comment:
if the dir is empty at the end of this method, it would be good to delete the dir itself